### PR TITLE
Show test execution duration

### PR DESCRIPTION
**Pull request by liamnichols**
Looks like the latest beta now includes the duration of the tests so i've added support :) 
The `time` attribute is now present on both the test case and suite in the junit ouput 🎉 

I also had to regenerate the `Valid1.plist` and `Valid2.plist` files from my own Xcode project so i've attached a copy incase anybody else needs to re-generate them in the future... Not sure if that should go into the repo though?

I've never really written any ruby before but the changes are minimal so it wasn't too hard but you might just want to sense check.. Tests all pass though.. Enjoy!

[Unit.xcodeproj.zip](https://github.com/KrauseFx/trainer/files/439669/Unit.xcodeproj.zip)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Prettify the `xcodebuild` output | :white_check_mark: | :no_entry_sign:
 Generate JUnit reports | :white_check_mark: | :white_check_mark:
 Generate HTML reports | :white_check_mark: | :no_entry_sign:
 Works when the `xcodebuild` output format changed | :no_entry_sign: | :white_check_mark:
-Show test execution duration | :white_check_mark: | :no_entry_sign:
+Show test execution duration | :white_check_mark: | :white_check_mark:
```
Comment (KrauseFx) [inline README.md]: Ha, that's awesome 🎉

**Comment by KrauseFx** [comment]
Hey @liamnichols, great catch, thanks for your contribution. Could you run `rubocop -a` to fix some of the code styling issues? 👍

**Comment by liamnichols** [comment]
```
➜  trainer git:(feautre/duration) ✗ rubocop   
Inspecting 15 files
...............

15 files inspected, no offenses detected
```

Done 👌

**Comment by KrauseFx** [comment]
Thanks for updating the PR, reviewing now 👍

**Comment by KrauseFx** [comment]
This is great, I'm preparing a new release right now 👍

**Pull request merged**